package-lock.json

```diff
@@ -1301,7 +1301,7 @@
     },
     "event-stream": {
       "version": "3.3.4",
-      "resolved": "https://registry.npmjs.org/event-stream/-/event-stream-3.3.4.tgz",
+      "resolved": "http://registry.npmjs.org/event-stream/-/event-stream-3.3.4.tgz",
```

Comment:
How come this changed?

Reply:
I've got no idea - I'll push an updated version